### PR TITLE
Fixes run energy, aggressiveness, and highscores

### LINKED ISSUE #131: Stop admin+ highscores saving
Simple just not at pc

### LINKED ISSUE #132: It seems like NPCs in wildy aren't aggressive.
Strange because we've got this already:

NpcAggressive.java (line 26)
		if (NpcHandler.npcs[i].inWild() && NpcHandler.npcs[i].MaxHP > 0) {
			return true;
		}

Maybe the public boolean inWild() in NPC.java isn't working properly.

### LINKED ISSUE #135: Run energy can restore to 101% instead of 100%.
Title

### PR DESCRIPTION
**Pull request by mrextremez**
closes #135 
closes #132 
closes #131

### CODE REVIEW

**Comment by Gptaqbc** [comment]
Siiiiick! :)

**Comment by Gptaqbc** [comment]
> Fixes npc aggressiveness
Does that fix aggro in wildy?

File: 2006Redone Server/src/redone/net/packets/impl/Commands.java [hunk 1]
```diff
@@ -684,7 +684,7 @@ public static void developerCommands(Client player, String playerCommand, String
                                         PlayerHandler.updateAnnounced = false;
                                         PlayerHandler.updateRunning = true;
                                         PlayerHandler.updateStartTime = System.currentTimeMillis();
-                                } catch (Exception ಠ_ಠ) {}
+                                } catch (Exception e) {}
```
Comment (dginovker) [inline 2006Redone Server/src/redone/net/packets/impl/Commands.java]: Lmao I didn't even know this was possible

**Comment by mrextremez** [comment]
> > Fixes npc aggressiveness
> 
> Does that fix aggro in wildy?

Yeah it fixes that too

**Pull request merged**